strategy/live_exec.py

```
@@ -160,15 +204,638 @@ def quote(condition_id: str, price: float, size: float, live: bool) -> None:
+            for item in items:
+                asset = item.get("asset_id")
+                event_type = item.get("event_type")
+                if event_type == "book" or asset == curr:
+                    last_delta_event["ts_recv"] = time.perf_counter_ns()
+                    last_delta_event["data"] = item
+                if asset == comp or (event_type == "book" and comp):
+                    bids = item.get("bids") or []
+                    if bids:
+                        comp_best_bid[0] = max(float(b.get("price", 0)) for b in bids)
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)